users/models.py

```diff
@@ -68,6 +68,8 @@ class User(AbstractUser):
     username: None = None
     email = models.EmailField(unique=True)
     last_activity = models.DateTimeField(auto_now_add=True, db_index=True)
+    create_post: bool = models.BooleanField(default=True)
+    warnings: int = models.IntegerField(default=0)
```

Comment:
What is this field for? Help text needed.

Reply:
Nope? Why nope? Help text will increase readability.

Reply:
so leave it here